tests/examples/step-49.diff

```diff
@@ -1,4 +1,28 @@
-131c131
-<   std::ifstream f("example.msh");
+29,31d28
+< #include <deal.II/grid/tria.h>
+< #include <deal.II/grid/tria_accessor.h>
+< #include <deal.II/grid/tria_iterator.h>
+32a30,31
+> #include <deal.II/grid/grid_in.h>
+> #include <deal.II/grid/grid_out.h>
+35,36c34,36
+< #include <deal.II/grid/grid_out.h>
+< #include <deal.II/grid/grid_in.h>
 ---
->   std::ifstream f("../../../source/step-49/example.msh");
+> #include <deal.II/grid/tria.h>
+> #include <deal.II/grid/tria_accessor.h>
+> #include <deal.II/grid/tria_iterator.h>
+38d37
+< #include <iostream>
+40c39
+< 
+---
+> #include <iostream>
+134c133
+<     std::ifstream f("example.msh");
+---
+>     std::ifstream f("../../../source/step-49/example.msh");
```

Comment:
```suggestion
```
I don't think we should change this.

Reply:
That was not a change I made, it was like that in the previous diff file as well.

I think the test for step-49 fails because of this, but I'm not sure.

```
      Start 47: examples/step-49.release
47: Test command: /usr/bin/cmake "-DTRGT=examples.step-49.release.test" "-DTEST=examples/step-49.release" "-DEXPECT=PASSED" "-DBINARY_DIR=/__w/dealii/dealii/build/tests/examples" "-P" "/__w/dealii/dealii/build/share/deal.II/scripts/run_test.cmake"
47: Test timeout computed to be: 600
47: gmake[3]: *** [binary/CMakeFiles/examples.step-49.release.test.dir/build.make:83: binary/step-49.release/serial/output] Error 1
47: gmake[2]: *** [CMakeFiles/Makefile2:2041: binary/CMakeFiles/examples.step-49.release.test.dir/all] Error 2
47: gmake[1]: *** [CMakeFiles/Makefile2:2048: binary/CMakeFiles/examples.step-49.release.test.dir/rule] Error 2
47: gmake: *** [Makefile:1018: examples.step-49.release.test] Error 2
47: Test examples/step-49.release: RUN
47: ===============================   OUTPUT BEGIN  ===============================
47: [  0%] Patching /__w/dealii/dealii/build/tests/examples/source/step-49.cc
47: patching file /__w/dealii/dealii/build/tests/examples/source/step-49.cc (read from /__w/dealii/dealii/tests/examples/../../examples/step-49/step-49.cc)
47: [  0%] Building CXX object binary/CMakeFiles/examples.step-49.release.dir/step-49.cc.o
47: [100%] Linking CXX executable step-49.release/step-49.release
47: [100%] Built target examples.step-49.release
47: [100%] Normalizing test output file /__w/dealii/dealii/build/tests/examples/binary/step-49.release/serial/output
47: examples/step-49.release: BUILD successful.
47: examples/step-49.release: RUN failed. ------ Return code 1
47: examples/step-49.release: RUN failed. ------ Result: /__w/dealii/dealii/build/tests/examples/binary/step-49.release/serial/failing_output
47: examples/step-49.release: RUN failed. ------ Partial output:
47: 
47: 
47: ----------------------------------------------------
47: Exception on processing: 
47: 
47: --------------------------------------------------------
47: An error occurred in line <2008> of file </__w/dealii/dealii/source/grid/grid_in.cc> in function
47:     void dealii::GridIn<dim, spacedim>::read_msh(std::istream&) [with int dim = 2; int spacedim = 2; std::istream = std::basic_istream<char>]
47: The violated condition was: 
47:     in.fail() == false
47: Additional information: 
47:     An input/output error has occurred. There are a number of reasons why
47:     this may be happening, both for reading and writing operations.
47:     
47:     If this happens during an operation that tries to read data: First,
47:     you may be trying to read from a file that doesn't exist or that is
47:     not readable given its file permissions. Second, deal.II uses this
47:     error at times if it tries to read information from a file but where
47:     the information in the file does not correspond to the expected
47:     format. An example would be a truncated file, or a mesh file that
47:     contains not only sections that describe the vertices and cells, but
47:     also sections for additional data that deal.II does not understand.
47:     
47:     If this happens during an operation that tries to write data: you may
47:     be trying to write to a file to which file or directory permissions do
47:     not allow you to write. A typical example is where you specify an
47:     output file in a directory that does not exist.
47: 
47: Stacktrace:
47: -----------
47: #0  /__w/dealii/dealii/build/lib/libdeal_II.so.9.7.0-pre: dealii::GridIn<2, 2>::read_msh(std::istream&)
47: #1  /__w/dealii/dealii/build/tests/examples/binary/step-49.release/step-49.release: Step49::grid_1()
47: #2  /__w/dealii/dealii/build/tests/examples/binary/step-49.release/step-49.release: main
47: --------------------------------------------------------
47: 
47: Aborting!
47: ----------------------------------------------------
47: 
47: 
47: examples/step-49.release: ******    RUN failed    *******
47: 
47: ===============================    OUTPUT END   ===============================
47: Expected stage PASSED - aborting
47: CMake Error at /__w/dealii/dealii/build/share/deal.II/scripts/run_test.cmake:115 (message):
47:   *** abort
```

Should we consider reverting it, or is there a better way to handle this?

Reply:
Sure, reverting is fine if that makes it work.